Cargo.toml

```diff
@@ -17,28 +17,28 @@ json = ["serde_json"]
 yaml = ["serde_yaml"]
 parse-value = ["pear"]
 test = ["tempfile", "parking_lot"]
-# toml = ["toml"]
+# toml = ["toml"] -> usable once we can select the dependency as dep:toml
```

Comment:
Could elaborate on that? I don't quit understand.

Reply:
One can not have a depending crate and feature with the same name, the commit was already there so I extended it. There is a feature in nightly where that will be allowed. You could then have a feature `toml` as well as a crate `toml`, declare the feature and use `dep:toml` to declare the dependency of that feature on the crate